### ISSUE TITLE
fetal pericardial effusion

### ISSUE THREAD

**Issue by sparkste** [obophenotype/human-phenotype-ontology#7008]
**Preferred term label:**
fetal pericardial effusion

**Synonyms**
fetal pericardial fluid

**Definition (free text, please give PubMed ID)**
abnormal fluid around the fetal heart

**Parent term (use hpo.jax.org/app)**
N/A

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Noonan syndrome and other RASopathies
inborn errors of metabolism
lymphatic disorders
skeletal disorders
cardiovascular anomalies
hematologic disorders
many other single gene disorders leading to non-immune hydrops

**Your nano-attribution (ORCID)**
0000-0002-8593-2186

**Comment by pnrobinson** [comment]
@JeroenBreckpot, @kristenleppertmiller  @sparkste -- thanks so much for joining the HPO workshop. We are going to ask people to comment on issues so that ideally two or more specialists "approve" each new term. Can we provide a PMID for more background information?

**Comment by sparkste** [comment]
Happy to provide a PMID. Just to clarify, you would like a PMID for fetal pericardial effusion specifically?



Teresa Sparks, MD, MAS

Assistant Professor

Maternal Fetal Medicine and Clinical Genetics

Department of Obstetrics, Gynecology, & Reproductive Sciences

University of California, San Francisco

________________________________

**Comment by pnrobinson** [comment]
@sparkste  here is an example
https://hpo.jax.org/app/browse/term/HP:0033659
the pubmed ID has a bit more information about the feature as well as an image. The basic idea is that we want to give people a place to go to find out more about each feature. If there is a good review article, that is ideal. Otherwise, a case report with an image or other explanation of the feature is a good choice.
Thanks!

**Comment by sparkste** [comment]
Thanks so much - makes sense. Here is a PMID that I think works well for fetal pericardial effusion. It has an overview of different etiologies/complications as well as images.
Appreciated,
Teresa



Teresa Sparks, MD, MAS

Assistant Professor

Maternal Fetal Medicine and Clinical Genetics

Department of Obstetrics, Gynecology, & Reproductive Sciences

University of California, San Francisco

________________________________

**Comment by sparkste** [comment]
PMID: 16098320



Teresa Sparks, MD, MAS

Assistant Professor

Maternal Fetal Medicine and Clinical Genetics

Department of Obstetrics, Gynecology, & Reproductive Sciences

University of California, San Francisco

________________________________

**Comment by pnrobinson** [comment]
added
